### PR TITLE
Write specified columns

### PR DESCRIPTION
**Pull request by ChengyuanZhao**
Write specified columns

### CODE REVIEW

File: spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java [hunk 1]
```diff
@@ -169,12 +171,44 @@
...
+	 * @param object the object to update.
+	 * @param includeColumns the columns to upsert. if none are given then all columns are
+	 * updated.
+	 */
+	void update(Object object, String... includeColumns);
+
+	/**
+	 * Update an object in storage.
+	 * @param object the object to update.
+	 * @param includeColumns the columns to update. if none are given then all columns are
```
Comment (balopat) [inline spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java]: nit: be more precise describing the includeColumns semantics.

File: spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java [hunk 2]
```diff
@@ -169,12 +171,44 @@
 	 */
 	void update(Object object);
 
+	/**
+	 * Update an object in storage.
+	 * @param object the object to update.
+	 * @param includeColumns the columns to upsert. if none are given then all columns are
```
Comment (balopat) [inline spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java]: nit: be more precise describing the includeColumns semantics.

File: spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java [hunk 3]
```diff
@@ -169,12 +171,44 @@
...
+	 */
+	void upsert(Object object, String... includeColumns);
+
+	/**
+	 * Update or insert an object into storage.
+	 * @param object the object to update or insert.
+	 * @param includeColumns the columns to upsert. if none are given then all columns are
+	 * upserted.
+	 */
+	void upsert(Object object, Optional<Set<String>> includeColumns);
```
Comment (balopat) [inline spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java]: nit: be more precise describing the includeColumns semantics.

File: spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java (hunk 2 above)
Comment (ChengyuanZhao) [reply spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java]: @balopat Thanks! added a commit to address this, but ironically that made the LGTM stale. PTAL again haha

**Pull request merged**